Doc/library/threading.rst

```diff
@@ -6,8 +6,6 @@
 
 **Source code:** :source:`Lib/threading.py`
 
---------------
-
```

Comment:
This was added on purpose, in all docs that use the `Source code:` line. Please restore, and in general be more cautious about removing things 🙂

Reply:
it's not removed but moved, it's still there, above Introduction.
should I add one or move the existing one?

Reply:
Keep it here!

Reply:
:+1: moved it back